src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/xlf/VS.LSClientRazor.Resources.de.xlf

```diff
@@ -12,6 +12,11 @@
         <target state="translated">Näherungsausdrücke werden ermittelt...</target>
         <note />
       </trans-unit>
+      <trans-unit id="View Code">
```

Comment:
I wonder if there's a way to exclude all of our .xlf files from GitHub diffs

Reply:
There is, yes. Will add it in.